src/components/tabController/apis/TabBar.api.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "TabController.TabBar",
```

Comment:
The name of the **file** should be with lowercase

Reply:
fixed